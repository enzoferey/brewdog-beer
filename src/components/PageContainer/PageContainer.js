import React from "react";
import PropTypes from "prop-types";
import style from "./page-container.scss";

const PageContainer = ({ children }) => (
  <div className={style.main}>{children}</div>
);

PageContainer.propTypes = {
  children: PropTypes.oneOfType([
    PropTypes.node,
    PropTypes.arrayOf(PropTypes.node),
  ]),
};

export default PageContainer;
