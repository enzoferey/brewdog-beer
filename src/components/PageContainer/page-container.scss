.main {
  max-width: 1280px;
  width: 90%;
  margin: 0 auto;
  background-color: white;
  padding-bottom: 100px;
}
