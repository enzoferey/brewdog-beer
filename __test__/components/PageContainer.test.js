import React from "react";
import PageContainer from "components/PageContainer";

const props = {
  background: "pink",
};

test("renders correctly", () => {
  testSnapshot(<PageContainer />);
});

test("applies `background` prop correctly", () => {
  testSnapshot(<PageContainer {...props} />);
});
