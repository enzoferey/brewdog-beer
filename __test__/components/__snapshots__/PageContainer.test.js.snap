// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`applies \`background\` prop correctly 1`] = `
<div
  style={
    Object {
      "backgroundColor": "pink",
    }
  }
>
  <div />
</div>
`;

exports[`renders correctly 1`] = `
<div
  style={
    Object {
      "backgroundColor": "white",
    }
  }
>
  <div />
</div>
`;
